api/features/views.py

```diff
@@ -59,15 +62,42 @@ def get_serializer_class(self):
             "create": ListCreateFeatureSerializer,
             "update": UpdateFeatureSerializer,
             "partial_update": UpdateFeatureSerializer,
-        }.get(self.action, FeatureSerializer)
+        }.get(self.action, ProjectFeatureSerializer)
+
+    @swagger_auto_schema(
+        request_body=UserIdsSerializer,
+        responses={200: ProjectFeatureSerializer},
+    )
+    @action(detail=True, methods=["POST"], url_path="add-owners")
+    def add_owners(self, request, *args, **kwargs):
+        feature = self.get_object()
+        try:
+            feature.add_owners_by_id(request.data["user_ids"])
```

Comment:
Can we move this to the serializer (one that inherits from the `UserIdsSerializer` class perhaps)? At the moment, this line will cause the API to throw a 500 if the client sends some bogus data (because we're not validating it anywhere).